Backend/src/test/java/com/onbank/ftp/FtpClientTest.java

```diff
@@ -0,0 +1,69 @@
+package com.onbank.ftp;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockftpserver.fake.FakeFtpServer;
+import org.mockftpserver.fake.UserAccount;
+import org.mockftpserver.fake.filesystem.DirectoryEntry;
+import org.mockftpserver.fake.filesystem.FileEntry;
+import org.mockftpserver.fake.filesystem.FileSystem;
+import org.mockftpserver.fake.filesystem.UnixFakeFileSystem;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class FtpClientTest {
+
+    private static FtpConnection ftpConnection;
+
+    private static FakeFtpServer fakeFtpServer;
+
+    @BeforeAll
+    static void setup() throws IOException {
+        fakeFtpServer = new FakeFtpServer();
+        fakeFtpServer.setServerControlPort(0);
+        fakeFtpServer.addUserAccount(new UserAccount("user", "password", "/data"));
+
+        FileSystem fileSystem = new UnixFakeFileSystem();
+        fileSystem.add(new DirectoryEntry("/data"));
+        fileSystem.add(new FileEntry("/data/foobar.txt", "abcdef 1234567890"));
+        fakeFtpServer.setFileSystem(fileSystem);
+        fakeFtpServer.start();
+
+        ftpConnection = new FtpConnection();
+        ftpConnection.setServer("localhost");
```

Comment:
skoro mamy już propertisy w testach to je wykorzystajmy zamiast wpisywać samemu wartości